Database/Corrections/sodQuestFixes.lua

```diff
@@ -5697,6 +5697,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -439770,
             [questKeys.zoneOrSort] = sortKeys.DRUID,
         },
+        [90309] = {
+            [questKeys.name] = "Tree of Life",
+            [questKeys.startedBy] = {{228596}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 50,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.DRUID,
+            [questKeys.objectivesText] = {"Talk to the Vengeful Wisp, and move to the middle of the area until you get the 'Shimmering Light' buff, talk to the wisp again and it will put Gift of the Wisp on the ground, loot it to receive the rune."},
```

Comment:
```suggestion
            [questKeys.objectivesText] = {"Talk to the Vengeful Wisp and kill Satyrs in Jadefire Run until you get the 'Shimmering Light' buff. Then talk to the wisp again and it will put Gift of the Wisp on the ground, loot it to receive the rune."},
```